python/sglang/srt/models/deepseek_v2.py

```diff
@@ -960,7 +986,7 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
                     hasattr(self_attn.kv_b_proj, "weight_scale")
                     and self_attn.w_scale is None
                 ):
-                    self_attn.w_scale = self_attn.kv_b_proj.weight_scale
+                    self_attn.w_scale = self_attn.kv_b_proj.weight_scale * 2.0
```

Comment:
Why multiply by 2 here?

Reply:
same as `* 2.0` in `normalize_e4m3fn_to_e4m3fnuz`. AMD FP8 value is 1/2 of that on OCP/NVIDIA, for the same byte code.

Reply:
But we also use this for NVIDIA GPU, do you need to use is_hip?

Reply:
let me update this.